packages/block-library/src/column/deprecated.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+/**
+ * WordPress dependencies
+ */
+import { InnerBlocks } from '@wordpress/block-editor';
+
+const deprecated = [
+	{
+		attributes: {
+			verticalAlignment: {
+				type: 'string',
+			},
+			width: {
+				type: 'number',
+				min: 0,
+				max: 100,
+			},
+		},
+		migrate( attributes ) {
+			return {
+				...attributes,
+				width: `${ attributes.width }%`,
```

Comment:
@tellthemachines, @ntsekouras - this one might need `isEligible` check. It's very likely we can avoid adding another deprecation in #26757.